readme.md

```diff
@@ -13,7 +13,7 @@ $ npm install --save is-binary-path
 ## Usage
 
 ```js
-var isBinaryPath = require('is-binary-path');
+const isBinaryPath = require('is-binary-path');
```

Comment:
Certainly. I intentionally kept unrelated changes like those outside of this PR. You want me to add `xo` too?